community_images/fluent-bit/bitnami/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '2'
+
+services:
+  fluent-bit:
+    image: ${FLUENT_BIT_IMAGE_REPOSITORY}:${FLUENT_BIT_IMAGE_TAG}
+    cap_add:
+      - SYS_PTRACE
+    ports:
+      - '2020:2020'
```

Comment:
No distinct port mapping allowed, please refer other images and see how we're getting port numbers during runtime in dc_coverage scripts.